mcs/class/System.XML/Test/System.Xml/XmlUrlResolverTests.cs

```diff
@@ -97,6 +97,7 @@ public void NullAbsoluteUriWithCustomSchemedRelativeUri ()
 		}
 
 		[Test]
+		[Category ("StaticNotWorking")] // Can't find .dll files when bundled in .exe

```

Comment:
The assemblies are being semantically statically linked together, and the AOTed assemblies are being statically linked, and the runtine is being statically linked into that. So everything is static. I can make it "StaticLinkNotSupported" or "StaticAotNotSupported" if either of those seem better to you. It's unfortunate that the dynamically linked​ "mobile static" profile was named such.